src/main/java/io/xstefank/wildfly/bot/PullRequestFormatProcessor.java

```diff
@@ -60,6 +61,12 @@ void verifyFormat(@PullRequest.Edited @PullRequest.Opened @PullRequest.Synchroni
         GHPullRequest pullRequest = pullRequestPayload.getPullRequest();
         Map<String, String> errors = new HashMap<>();
 
+        if (Pattern.compile("@%s skip format".formatted(wildFlyBotConfig.githubName()), Pattern.DOTALL | Pattern.CASE_INSENSITIVE | Pattern.LITERAL)
```

Comment:
move pattern compile to field, we don't need to recompile the pattern on every invocation

Reply:
isn't the case insensitive possible issue with `@bot-nAmE` mentions? I believe that `@mention` is not case insensitive.

Reply:
true but I wanted to do is just as temporary thing anyways. I can remove the insensitivity if you want